upower: add page (#5923)
tldr-pages/tldr: pages/linux/upower.md

# upower

> > System Utility to provide battery information and statistics.

Reviewer: ```suggestion
> System utility to provide battery information and statistics.
```

> > System utility to provide battery information and statistics.

Reviewer 2: I think we should change this too. Either:
```suggestion
> System utility to provide power and battery information and statistics.
```
or 
```suggestion
> System utility to provide power information and statistics.
```

Author: Commited option 1

> > More information: <https://upower.freedesktop.org/>.

Reviewer 2: ```suggestion
> More information: <https://upower.freedesktop.org/docs/upower.1.html>.
```
I think we should directly point to the man page. Finding the home page from there is easier than the other way around.

Reviewer 2: We only use `manned.org` if there isn't an official source.

Reviewer: Actually, manned.org looks better, and since they have exactly the same content, I think we should use manned.org.  wdyt?

Author: The official page isn't just a man page, it is more

Reviewer: Then let's use the official site 👍🏻

> - Dump power and battery information:

Author: Good change!

`upower --dump`

> - List connected battery-using devices:

Reviewer 2: I'm not really happy with this description. When I run that command, I get:
```
.../line_power_AC0
.../battery_BAT0
.../DisplayDevice
```
I wouldn't call `line power` and `battery` battery-using. However, there isn't really good documentation for this and the only description I found was `enumerate all power objects on the system`. 
How about:
```suggestion
- List all power objects on the system:
```

Reviewer 2: That's fine too.

`upower --enumerate`

> - Listen for changes in battery status:
> 
> `upower --monitor`
> 
> - Watch for and print battery status changes:

Reviewer 2: Since both examples are almost the same I would also phrase them the same.
```suggestion
- Watch for and print battery status changes:

`upower --monitor`

- Watch for and print detailed battery status changes:
```

`upower --monitor-detail`

> - Show version of software:

Reviewer: ```suggestion
- Display version:
```
